segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/AppendProcessor.java

```diff
@@ -265,6 +276,9 @@ private void handleException(long requestId, String segment, String doingWhat, T
         } else if (u instanceof WrongHostException) {
             WrongHostException wrongHost = (WrongHostException) u;
             connection.send(new WrongHost(requestId, wrongHost.getStreamSegmentName(), wrongHost.getCorrectHost()));
+        } else if (u instanceof BadAttributeUpdateException) {
```

Comment:
you use `BadAttributeUpdateException` here; do you also want to include the `BadOffsetException` too?